pkg/arvo/mar/publish/action.hoon

```diff
@@ -69,7 +69,7 @@
       %-  ot:dejs
       :~  who+(su:dejs fed:ag)
           coll+so:dejs
-          name+(su:dejs sym)
```

Comment:
If I understand correctly, this is `post` in the `action:publish` type. Not sure why we use `name` here.

Regardless, publish [uses that as a `@tas`](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/publish.hoon#L1139), so accepting technically-not-valid-term strings into it might be ill-advised. (See also @joemfb's comment.)

That said, @ixv has already given his blessing, and publish will relaunch with all new code in the near future anyway, so this is probably fine.

Reply:
Note that posts are already saved to clay with `@da`-style id:s when their titles start with numbers, which is why in the mark for new-post 9 lines above `name` is parsed with `so`. 

I do agree that it's incorrect, but invalid `@tas` post id:s are already accepted regardless of this PR being merged or not. You just can't comment on them.

Of course none of this matters very much with what I hear of the publish rewrite, I'm fine with you guys closing this or merging this as you wish!